frontend/gateway/gateway.go

```diff
@@ -63,7 +63,7 @@ import (
 )
 
 const (
-	keySource = "source"
+	KeySource = "source"
```

Comment:
Is this const made public so it can be used downstream? If so, it should first be moved to another package to avoid unnecessary imports similar to https://github.com/moby/buildkit/blob/master/exporter/exptypes/keys.go

Reply:
This seems to be all internal impports so not a big issue. If you want can move it to frontend pkg (or errdefs) that I think are already imported.

Reply:
I don't mind duplicating the key if you guys think that's preferable.

Reply:
I've moved it to `frontend`,  PTAL